### DISCUSSION TITLE
How should a lesson proceed if a teacher for a session is away ( illness, VAB, etc ) without warning?

### DISCUSSION THREAD

**Discussion by mahesh-panchal** [NBISweden/Training-Tech-shorts#31] [Q&A]
If someone is designated to teach a lesson, but is unable to make it, for example, due to illness, what should happen during that session?

**Comment by richelbilderbeek** [comment]
I feel it would be OK to cancel that session. If there are already people, they can decide what to do: leave or discuss something unprepared.

**Reply by CormacKinsella** [reply]
I agree. If we can notify in advance that we won't make it, maybe there is time for someone to jump in with a new topic, or we if not we could cancel it.
